docs/faqs.md

```diff
@@ -65,11 +67,12 @@ Boom, you have a new follower!
 If you ever want to add Listings locally, you must have credits on your account
 to "pay" for listing. Here's how:
 
-1. Open the rails console `rails c`.
+1. Open the rails console `rails console`.
```

Comment:
The actual fix in the docs is here. Everything above that changed is prettier formatting in action.